aiidalab_widgets_base/computational_resources.py

```diff
@@ -225,12 +245,56 @@ def _setup_new_code(self, _=None):
                 }
 
 
+class SshConnectionState(enum.Enum):
```

Comment:
I had not noticed that the `SshConnectionState` and the `SshPossibleRepsonses` from below are intimately linked via their order. They must therefore be part of the same data structure to avoid getting out of sync. Or at the very least there must be a comment that makes this intimate connection clear.

Reply:
I add a commend for `SshPossibleRepsonses` and convert idx return from `spawn.expect` to `Enum` class and compare it by identity.